aws/DRVault-Single-Deployment.yaml

```diff
@@ -395,14 +395,30 @@ Resources:
 
           def handler(event, context):
             try:
-              response = boto3.client('ec2').describe_images(
-                  Owners=['138339392836', '284566481789', '923248178732', '604515549189'],
-                  Filters=[
-                    {'Name': 'tag:Cyberark_version', 'Values': [event['ResourceProperties']['Version']]},
-                    {'Name': 'tag:component', 'Values': [event['ResourceProperties']['Component']]},
-                    {'Name': 'tag:os', 'Values': [event['ResourceProperties']['OS']]},
-                  ],
-              )
+              ver= event['ResourceProperties']['Version']
+              major= int(ver.split(".")[0])
```

Comment:
Consider renaming to `pasVerMajor`

Also - what if the `version` is not of `XX.x` ? is it possible at all?
In any case - defensive code is usually better - your call

Reply:
This is not possible by our tagging standard (even if we publish PTA 12.2.3, the tag remains 12.2), if this is what you mean